Dockerfile

```diff
@@ -16,29 +16,33 @@ RUN mkdir -p /usr/share/man/man1 \
   zip \
   make \
   ca-certificates \
-  && curl -fsSL https://deb.nodesource.com/setup_20.x | bash - \
+  && curl -fsSL https://deb.nodesource.com/setup_24.x | bash - \
   && apt-get install --no-install-recommends -y nodejs \
-  && apt-get install -y --no-install-recommends nuget libgtk2.0-0 libgtk-3-0 libgbm-dev libnotify-dev libgconf-2-4 libnss3 libxss1 libasound2 libxtst6 xauth xvfb procps\
   && apt-get clean && rm -rf /var/lib/apt/lists/*
 
 # install modern version of java
 RUN apt-get update \
-  && apt-get install --no-install-recommends -y openjdk-17-jdk openjdk-17-jre \
+  && apt-get install --no-install-recommends -y openjdk-25-jdk openjdk-25-jre \
   && apt-get clean && rm -rf /var/lib/apt/lists/*
 
 # install docker-compose
-RUN curl -L "https://github.com/docker/compose/releases/download/v2.38.2/docker-compose-$(uname -s)-$(uname -m)" -o /usr/local/bin/docker-compose \
+RUN curl -L "https://github.com/docker/compose/releases/download/v2.40.3/docker-compose-$(uname -s)-$(uname -m)" -o /usr/local/bin/docker-compose \
   && chmod +x /usr/local/bin/docker-compose
 
+# install GUI libs for headless browser testing
+RUN apt-get update \
+  && apt-get install -y --no-install-recommends libgtk2.0-0 libgtk-3-0 libgbm-dev libnotify-dev libnss3 libxss1 libxtst6 xauth xvfb procps \
+  && apt-get clean && rm -rf /var/lib/apt/lists/*
+
 # install Chromium for (unit)-testing during build-phase
-RUN apt-get update && \
-  apt-get install -y --no-install-recommends chromium && \
-  rm -rf /var/lib/apt/lists/*
+RUN apt-get update \
+  && apt-get install -y --no-install-recommends chromium \
+  && rm -rf /var/lib/apt/lists/*
 
 # install Firefox for (unit)-testing during build-phase
-RUN apt-get update && \
-  apt-get install -y --no-install-recommends firefox-esr && \
-  rm -rf /var/lib/apt/lists/*
+# RUN apt-get update \
+#   && apt-get install -y --no-install-recommends firefox-esr \
```

Comment:
@Mastermindzh The same for Firefox-esr. I did find a post that explained how to add the Mozilla package repo, but I don't know if that was a correct way to do it. It is also available as snap package

Reply:
You can also install the regular `firefox` package instead of the esr one ;)